shards/tests/network-ws.shs

```diff
@@ -1,12 +1,17 @@
 ; Run the same network test except use Network.WS client & server
-"network.shs" | FS.Read = base-script
-Regex.Replace("Network.Client\\(\".*\" [0-9]+" "Network.WS.Client(\"ws://127.0.0.1:9191\"")
-Regex.Replace("Network.Server\\(" "Network.WS.Server(") = new-script
-Shards.Read | Shards.Distill("runner") = runner
-WireRunner(runner)
-
 @mesh(root)
 
+@wire(main-test {
```

Comment:
Why can't this run on the script root anymore, WireRunner doing something?

Reply:
should be ok actually, will try revert